include/eve/arch/cpu/wide.hpp

```diff
@@ -346,10 +346,10 @@ namespace eve
     void swap( wide& other ) { std::swap(this->storage(), other.storage()); }
 
     //! Pre-incrementation operator
-    EVE_FORCEINLINE wide &operator++()    noexcept  { return detail::self_add(*this, wide{1}); }
+    EVE_FORCEINLINE wide &operator++()    noexcept  { return *this += wide{1}; }
 
     //! Pre-decrementation operator
-    EVE_FORCEINLINE wide &operator--()    noexcept  { return detail::self_sub(*this, wide{1}); }
+    EVE_FORCEINLINE wide &operator--()    noexcept  { return *this -= wide{1}; }
```

Comment:
Well. Hmm. I think that for floats operator++ goes to the next float.
Not sure. If you use operator++ on floats it's on you anyways)

Reply:
++ just adds 1